bin/manage.py

```diff
@@ -16,7 +16,7 @@
 from pymongo import MongoClient
 from pymongo.errors import *
 
-consul = pyconsul.Consul(host=os.environ.get('CONSUL', 'consul'))
+consul = pyconsul.Consul(host='localhost')
```

Comment:
We will likely need to make the consul agent optional in the future, can you revert this change.